presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import java.util.Optional;
+
+public class ColumnStatistics
+{
+    public static final ColumnStatistics EMPTY_STATISTICS = ColumnStatistics.builder().build();
+
+    private final Optional<StatisticsValue<Long>> dataSize;
+    private final Optional<StatisticsValue<Long>> rowsCount;
+    private final Optional<StatisticsValue<Object>> min;
+    private final Optional<StatisticsValue<Object>> max;
```

Comment:
I started with just random few stats. Of course it will grow over time.
I have to check what hive supports out of the box and expose that for a start.